packages/bot/src/components/quick-strategy.jsx

```diff
@@ -0,0 +1,282 @@
+                                                        type='text'
+                                                        error={touched.size && errors.size}
+                                                        label={translate('Size')}
+                                                    />
+                                                )}
+                                            </Field>
+                                            <Field name='profit'>
+                                                {({ field }) => (
+                                                    <Input
+                                                        {...field}
+                                                        className='quick-strategy__input'
+                                                        type='text'
+                                                        error={touched.profit && errors.profit}
+                                                        label={translate('Maximum profit')}
+                                                    />
+                                                )}
+                                            </Field>
+                                        </div>
+                                        <div className='quick-strategy__form-footer' >
+                                            <Button
+                                                type='submit'
+                                                className={classnames(
+                                                    'btn--primary',
+                                                    'btn--primary--red',
+                                                    'quick-strategy__button--create',
+                                                )}
+                                                text={translate('Create')}
+                                                is_disabled={!isValid || isSubmitting}
+                                            />
+                                        </div>
+                                    </Form>
+                                )
+                            }
+                        </Formik>
+                    </div>
+                </div>
+            </Modal>
+        );
+    }
+}
+
+QuickStrategy.propTypes = {
+    active_index          : proptypes.number,
+    createStrategy        : proptypes.func,
+    initial_values        : proptypes.object,
+    is_strategy_modal_open: proptypes.bool,
+    market_dropdown       : proptypes.oneOfType([
+        proptypes.array,
+        proptypes.object,
+    ]),
+    onChangeMarketDropdown   : proptypes.func,
+    onChangeTradeTypeDropdown: proptypes.func,
+    onMount                  : proptypes.func,
+    setActiveTabIndex        : proptypes.func,
+    toggleStrategyModal      : proptypes.func,
+    trade_type_dropdown      : proptypes.oneOfType([
+        proptypes.array,
+        proptypes.object,
```

Comment:
- `proptypes` should be capitalised.
- Please check if we need multiple types for `trade_type_dropown` + `market_dropdown`